src/program/lwaftr/tests/subcommands/loadtest_test.py

```diff
@@ -0,0 +1,60 @@
+"""
+Test the "snabb lwaftr loadtest" subcommand. Needs NIC names.
+
+Since there are only two NIC names available in snabb-bot, and we need to
+execute two programs networked to each other ("run" and "loadtest"), they
+are set to on-a-stick mode, so that they use one NIC each instead of two.
+"""
+
+import unittest
+
+from lib import sh
+from lib.test_env import BENCHDATA_DIR, DATA_DIR, SNABB_CMD, nic_names
+
+
+SNABB_PCI0, SNABB_PCI1 = nic_names()
+
+
+@unittest.skipUnless(SNABB_PCI0 and SNABB_PCI1, 'NICs not configured')
+class TestLoadtest(unittest.TestCase):
+
+    run_cmd_args = (
+        SNABB_CMD, 'lwaftr', 'run',
+        '--bench-file', '/dev/null',
+        '--conf', DATA_DIR / 'icmp_on_fail.conf',
+        # NUMA node #0.
+        '--cpu', '4',
+        '--on-a-stick', SNABB_PCI1,
+    )
+
+    loadtest_cmd_args = (
+        SNABB_CMD, 'lwaftr', 'loadtest',
+        '--bench-file', '/dev/null',
+        # Something quick and easy.
+        '--program', 'ramp_up',
+        '--step', '0.1e8',
+        '--duration', '0.1',
+        '--bitrate', '0.2e8',
+        # NUMA node #1.
+        '--cpu', '10',
+        # Just one card for on-a-stick mode.
+        BENCHDATA_DIR / 'ipv4_and_ipv6_stick_imix.pcap', 'ALL', 'ALL', SNABB_PCI0,
```

Comment:
Weird indeed: it comes from the `snabb1` current config. Relevant excerpt from `snabb1`'s `/etc/nixos/configuration.nix`:
```
  services.snabb_bot.environment = ''
        export SNABB_PCI0=0000:82:00.0
        export SNABB_PCI1=0000:02:00.0
  '';
```
The `0000:82:00.0` NIC is connected to NUMA #1, and the `0000:82:00.0` NIC to NUMA #0. The env vars definition should probably be swapped.